client/app/visualizations/cohort/index.js

```diff
@@ -115,17 +115,21 @@ function prepareSimpleData(sortedData, options) {
 function prepareData(rawData, options) {
   rawData = _.map(rawData, item => ({
     date: item[options.dateColumn],
-    stage: item[options.stageColumn],
-    total: item[options.totalColumn],
-    value: item[options.valueColumn],
+    stage: parseInt(item[options.stageColumn], 10),
+    total: parseFloat(item[options.totalColumn]),
+    value: parseFloat(item[options.valueColumn]),
   }));
   const sortedData = _.sortBy(rawData, r => r.date + parseInt(r.stage, 10));
```

Comment:
`parseInt` not needed here (value already converted)

Reply:
👍 removed.